runtime/astar/src/lib.rs

```diff
@@ -1118,20 +1169,44 @@ pub type Executive = frame_executive::Executive<
 >;
 
 parameter_types! {
-    pub const DappStakingMigrationName: &'static str = "DappStakingMigration";
-
+    pub const StaticPriceProviderName: &'static str = "StaticPriceProvider";
 }
 
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
 pub type Migrations = (
     frame_support::migrations::RemovePallet<
-        DappStakingMigrationName,
+        StaticPriceProviderName,
         <Runtime as frame_system::Config>::DbWeight,
     >,
+    OracleIntegrationLogic,
+    pallet_price_aggregator::PriceAggregatorInitializer<Runtime, InitPrice>,
 );
 
+pub struct InitPrice;
+impl Get<CurrencyAmount> for InitPrice {
+    fn get() -> CurrencyAmount {
+        // 0.18 $
+        CurrencyAmount::from_rational(18, 100)
+    }
+}
```

Comment:
nitpick: why not use simply `parameter_types` for this?

Reply:
Good comment, I'll change it!